### ISSUE TITLE
What does CS mean?

### ISSUE THREAD

**Issue by theSage21** [PyJaipur/PyJaipur#69]
- This talk explores the concept of computation and how it impacts our lives on a daily basis.
- No need for a laptop.

**Comment by Shivank98** [comment]
Is it for some college? Sparsh?

**Comment by theSage21** [comment]
Yes. This is what is being proposed for the Amity event.

**Comment by theSage21** [comment]
@piratesparsh please mention here once things have been confirmed on your side. I can move this talk to the scheduled section in the [Talks project](https://github.com/PyJaipur/PyJaipur/projects/1#card-38336474)

**Comment by theSage21** [comment]
@piratesparsh putting this on hold until amity send us a confirmation.

**Comment by piratesparsh** [comment]
We can have it on saturday 11:00 A.m.

**Comment by theSage21** [comment]
It's a little tight but that should be do-able:

- Can you post the amity logo here so that we can use it in our posters?
- Please forward an official mail from hod or whoever is the next best person to pyjaipur.india@gmail.com regarding this event.

**Comment by theSage21** [comment]
Actually, @piratesparsh I'm not sure. I'd like to schedule this on a volunteer call before finalizing it.

**Comment by piratesparsh** [comment]
> It's a little tight but that should be do-able:
> 
> * Can you post the amity logo here so that we can use it in our posters?
> * Please forward an official mail from hod or whoever is the next best person to [pyjaipur.india@gmail.com](mailto:pyjaipur.india@gmail.com) regarding this event.

This won't be possible I guess, As far as I am aware, I will only be able to circulate the webinar link throughout the whole university via official noticeboard, hoping for any such mails would be a wastage of time.

**Comment by piratesparsh** [comment]
> Actually, @piratesparsh I'm not sure. I'd like to schedule this on a volunteer call before finalizing it.

**Comment by piratesparsh** [comment]
> Actually, @piratesparsh I'm not sure. I'd like to schedule this on a volunteer call before finalizing it.

Let me know the date from you too, if you have any preferences.

**Comment by theSage21** [comment]
Oh, so it's not an official Amity collaboration right? Then we can just schedule this talk like a normal pyjaipur meetup talk.

**Comment by piratesparsh** [comment]
I don't think you would be able to use Amity's or any other institution's logo, using brand logos require a more complex collaboration. I didn't saw any of the institutions whosoever conducted any seminar/workshop using Amity's logo anywhere. It solely depends on the university's concern.

**Comment by piratesparsh** [comment]
> Oh, so it's not an official Amity collaboration right? Then we can just schedule this talk like a normal pyjaipur meetup talk.

What does collaboration mean for just a seminar? We got permission for the seminar invite to be sent to everyone in the university, I am satisfied with that.  We must consider this as an opportunity to have our group PyJaipur attain some limelight in Amity University Rajashtan.  Rest relies on you.

**Comment by theSage21** [comment]
Let us discuss this on #103 . A few points I want to be considered are:

- If PyJ cannot mention amity in its posters what's the difference between a normal PyJ talk and this talk?
    - Essentially it's a normal PyJ talk with some external group coming and attending the call.
- AFAIK when a college student body announces an event, the event poster carries logos of both parties involved. Not sure if this has changed or if this is not followed in amity. I've seen it happen in the past for MUN events and so on.
- Collaboration for a seminar/event usually means that a speaker and topic are supplied by one party and venue and audience are supplied by another. Along with this, both parties propagate the event via their own marketing channels.
- As for gaining limelight in Amity University Rajasthan, I personally have no interest in any sort of limelight anywhere. I just want impact and change. But since that's my personal opinion, I'd like to discuss this on call with everyone present and decide if limelight in amity is required for PyJ or not.

**Comment by piratesparsh** [comment]
I am abiding on what I've promised, and I hereby request you to kindly refer to our previous conv.
You are senior and my experience has no par with yours, you may discuss with whomsoever you may feel relevant. 
As you stated above, _Talk's purpose is to create an impact and change_  then rest other things are of lesser concern.
At last, proceed with your own wisdom. :-)

**Comment by piratesparsh** [comment]
I don't want any talk at my university, I am pulling out. Let this a PyJtalk. Thank you for devoting your valuable time so far. I am not having any personal gains if the session happens or not, and Amity is not my brand.

**Comment by theSage21** [comment]
Hmm, I have very little experience with these kinds of situations so far. Maybe someone else will know how to handle this better. I'll do my best to address your points.

> I am abiding on what I've promised, and I hereby request you to kindly refer to our previous conv.
> You are senior and my experience has no par with yours, you may discuss with whomsoever you may feel relevant.
> As you stated above, _Talk's purpose is to create an impact and change_ then rest other things are of lesser concern.
> At last, proceed with your own wisdom. :-)

@piratesparsh I didn't mean to imply that there's something wrong with the work you've done or with the nature of how this talk is being organized. I wanted to know things so that we can design posters and promotional material correctly.

> I don't want any talk at my university, I am pulling out. Let this a PyJtalk. Thank you for devoting your valuable time so far. I am not having any personal gains if the session happens or not, and Amity is not my brand.

It is sad to hear that. Please attend the Sunday volunteer call in case you change your mind.

In case anyone else has a way to handle this situation please do comment below.
